### PR TITLE
test: `testID` added in `TabBar` for testing

### PR DESCRIPTION
**Pull request by Asad520**
**Motivation**

I was unable to scroll the tab bar and couldn't press on tabs that were not in the visible view as detox requires it to be a scrollable component. Adding a test id fixes it as we can grab the tabBar using `by.id(testID)`.

**Test plan**

I simply added a tabBarTestID prop in TabBar to grab the bar. Now it is scrollable in both Android & iOS

### CODE REVIEW

**Comment by okwasniewski** [comment]
Hey @Asad520,

Thanks for your PR! 

Right now there is an PR open to refactor all components to function based (#1392). To avoid conflicts I can get back to this once #1392 gets merged 👍🏻

File: README.md [hunk 1]
```diff
@@ -408,6 +408,10 @@ Function which takes an object with the current route and returns a test id for
 />
 ```
 
+##### `tabBarTestID`
```
Comment (satya164) [inline README.md:411]: Let's rename the prop to `testID`

Also, move this section towards the end. The documentation is organized to have functions together

Comment (Asad520) [reply README.md]: Thanks for pointing out, I've made the suggested changes.

**Comment by okwasniewski** [comment]
Hey @Asad520,

Please rebase your branch over main and resolve conflicts.

**Comment by Asad520** [comment]
@okwasniewski i've rebased the branch and resolved the conflicts  👍

**Review by okwasniewski — APPROVED** [review]

**Pull request merged**